frontend/templates/product/server.tsx

```diff
@@ -23,9 +23,11 @@ export const getServerSideProps: GetServerSideProps<ProductTemplateProps> =
       const { stores, ...otherLayoutProps } = await getLayoutServerSideProps({
          storeCode: DEFAULT_STORE_CODE,
       });
+      const ifixitOrigin = ifixitOriginFromHost(context);
```

Comment:
Minor: Now that we're storing the result of `ifixitOriginFromHost(context)` in a variable, we can use this variable in `pageProps.appProps` below instead of calling the function again